gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcWriteChannel.java

```diff
@@ -488,6 +481,10 @@ public void beforeStart(
           ClientCallStreamObserver<WriteObjectRequest> clientCallStreamObserver) {
         clientCallStreamObserver.setOnReadyHandler(ready::countDown);
       }
+
+      public boolean isComplete() {
```

Comment:
can this be not public?

Reply:
No, it has to be public as I need it to find the state of observer.